hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -23,6 +23,7 @@
     "hw/top_earlgrey/data/ip/chip_keymgr_testplan.hjson",
     "hw/top_earlgrey/data/ip/chip_lc_ctrl_testplan.hjson",
     "hw/top_earlgrey/data/ip/chip_kmac_testplan.hjson",
+    "hw/top_earlgrey/data/ip/chip_flash_ctrl_testplan.hjson",
```

Comment:
Could you insert the testplans in alphabetical order?

Reply:
Revised IP block specific top level test plan only